app/views/events/show.html.haml

```diff
@@ -17,9 +17,11 @@
         Maximum participant:
       = @event.capacity
 
+  - if current_user
```

Comment:
Does the booking_buttons partial only ever need to be shown to logged in users? If yes, consider moving the `if current_user` check into the `booking_buttons` partial.